pkg/kosmosctl/install/install.go

```diff
@@ -554,30 +554,31 @@ func (o *CommandInstallOptions) createOperator() error {
 }
 
 func (o *CommandInstallOptions) createControlCluster() error {
+
+	clusterArgs := []string{"cluster"}
+
 	switch o.Module {
 	case utils.ClusterLink:
+		joinOptions := join.CommandJoinOptions{
```

Comment:
looks good, can 'joinOptions' be further optimized? Because after it was proposed, it seemed that there were still some redundancies

Reply:
'joinOptions' as a whole, maybe well  not split again.

Reply:
extracting shared parameters will make the module features look clearer and reduce redundant parameters, what do you think?

Reply:
modified